app/javascript/src/utils/search.js

```diff
@@ -78,3 +78,25 @@ export function preloadSearchResults({
     )}`,
   );
 }
+
+/**
+ * A helper method to call /search endpoints.
+ *
+ * @param {string} endpoint - The search endpoint you want to request (i.e. tags).
+ * @param {object} dataHash - A hash with any params that need to be included in the request.
```

Comment:
The only thing that I'd add here is that the data hash should include whatever search params you're looking for, since that's important to know if you try to use this method.
```suggestion
 * @param {object} dataHash - A hash with the search params that need to be included in the request.
```